### PR TITLE
docs: add Zoom meeting schedule to README

### PR DESCRIPTION
**Pull request by andrewrynhard**
Signed-off-by: Andrew Rynhard <andrew@andrewrynhard.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,13 @@ Follow us on Twitter for the latest on Talos, or join our slack for in-depth dis
   </a>
 </p>
 
+Join our weekly Zoom meetings:
+
+https://zoom.us/j/3595189922
```
Comment (bradbeam) [inline README.md]: no cute zoom cname?

Comment (andrewrynhard) [reply README.md]: Ha, not yet. Wasn't sure if we should.

Comment (smira) [reply README.md]: I wonder if I missed today's meeting? I never saw the link... :confused:

Comment (andrewrynhard) [reply README.md]: That was my fault... I posted in the community channel but not the engineering channel.

**Pull request merged**